tracing/zipkin/README.md

```diff
@@ -36,131 +35,97 @@ The following services have been set-up to run:
 
 ## Middleware Usage
 
-Wrap a server- or client-side [endpoint][] so that it emits traces to a Zipkin
-collector. Make sure the host given to `MakeNewSpanFunc` resolves to an IP. If
-not your span will silently fail!
+Follow the [addsvc] example to check out how to wire the Zipkin Middleware. The
+changes should be relatively minor.
 
-[endpoint]: http://godoc.org/github.com/go-kit/kit/endpoint#Endpoint
+The [zipkin-go-opentracing] package has support for Kafka and Scribe collectors
+as well as using Go Kit's [Log] package for logging.
 
-If needing to create child spans in methods or calling another service from your
-service method, it is highly recommended to request a context parameter so you
-can transfer the needed metadata for traces across service boundaries.
+### Span per Node vs. Span per RPC
+OpenTracing tends to support the `span per node` model, meaning the server and
+client side of an RPC call are dealt with in separate spans. This diverts from
+the Zipkin V1 model which is more tailored towards the `span per RPC call` model
+in which the server and client side annotations of the same RPC call are part of
+the same span. The [zipkin-go-opentracing] implementation allows you to choose
+which model you wish to use. Make sure you select the same model consistently
+for all your services that are required to talk to each other or you will have
+trace propagation issues. If using non OpenTracing / legacy instrumentation use
+the `span per RPC call` model.
```

Comment:
I suspect new users won't totally understand the distinction here. Is there some linkable page on OpenTracing or Zipkin that can explain the difference?